backend/src/handlers/RelayPool/OutstandingDebtChanged.ts

```diff
@@ -28,11 +28,22 @@ export default async function ({
   context: Context<'RelayPool:OutstandingDebtChanged'>
 }) {
   // Extract the new debt value from the event arguments.
-  const { newDebt, origin, newOriginDebt } = event.args
+  const { newDebt, oldDebt, newOriginDebt, oldOriginDebt, origin } = event.args
 
   // Retrieve the pool using the contract address from the event log.
   const poolAddress = event.log.address
 
+  const pool = await context.db.find(relayPool, {
+    chainId: context.chain.id,
+    contractAddress: poolAddress,
+  })
+
+  if (pool!.outstandingDebt !== oldDebt) {
```

Comment:
Here could also check if the `pool.outstandingDebt` fetched directly from the contract is equal to the newDebt

Reply:
Let me actually clean that. I don't think that's useful in the backend itself.